scripts/import-update-key

```diff
@@ -10,11 +13,12 @@ if [ -z "$keyfile" ]; then
   exit 1
 fi
 
-keyhash=$(gpg $gpgopts --with-fingerprint $keyfile | sed -ne 's#^pub  [^ ]\+/\([^ ]\+\).*#\1#p')
-if [ ${#keyhash} -ne 8 ]; then
+keyinfo=$(gpg --show-keys $listopts "$keyfile" 2>/dev/null || gpg $listopts "$keyfile")
```

Comment:
The upstream manpage explicitly says to never use the output of these commands directly, but to instead use `--with-colons` which is meant to be machine parsable and stable. We should follow what the manpage says.